modAionImpl/src/org/aion/zero/impl/db/AionRepositoryDummy.java

```diff
@@ -139,24 +139,33 @@ public BigInteger getBalance(Address addr) {
     public ByteArrayWrapper getStorageValue(Address addr, ByteArrayWrapper key) {
         IContractDetails details = getContractDetails(addr);
         ByteArrayWrapper value = (details == null) ? null : details.get(key);
-        if (value == null) {
-            return null;
-        }
-        return (value.isZero()) ? null : value;
-    }
-
-    public void addStorageRow(Address addr, ByteArrayWrapper key, ByteArrayWrapper value) {
-        IContractDetails details = getContractDetails(addr);
 
-        if (details == null) {
-            createAccount(addr);
-            details = getContractDetails(addr);
+        if (value != null && value.isZero()) {
+            // TODO: remove when integrating the AVM
+            // used to ensure FVM correctness
+            throw new IllegalStateException(
+                    "The contract address "
+                            + addr.toString()
+                            + " returned a zero value for the key "
+                            + key.toString()
+                            + " which is not a valid stored value for the FVM. ");
         }
 
-        details.put(key, value);
-        detailsDB.put(ByteArrayWrapper.wrap(addr.toBytes()), details);
+        return value;
     }
 
+    // never used
```

Comment:
delete?

Reply:
We may want to remove the full class, cause there are multiple classes used for repository test functionality. But this feels like a separate task.

Reply:
If this is the test class that ends up remaining after refactoring, the functionality may be reenabled.